bundles/org.openhab.binding.tankerkoenig/README.md

```diff
@@ -180,7 +180,7 @@ Note: If the API-key is banned by Tankerkönig, the reason has to be cleared wit
 
 The correct usage of opening times needs the information if the actual day is a holiday.
 The binding expects a switch item linked to the Webservice channel holiday.
-This switch can be set either manually (only suggested for testing!), by a rule &lsqb;openHAB1-addons rules&rsqb; or by the usage of the CALDAV binding with a calendar.
+This switch can be set either manually (only suggested for testing!) or by a rule which uses the [ephemeris action](<https://www.openhab.org/docs/configuration/actions.html#ephemeris>) to set that switch.
```

Comment:
Are you sure the <> around the link are needed?

Reply:
Nope, it looked to me (on the preview) as if I needed them in order to show the link correctly. I could be wrong!

Reply:
I can override the DCO check, sign-off looks good to me.

Reply:
Thanks for overriding, but I still do not understand this one problem.